src/Stratis.Bitcoin.Features.Wallet/WalletManager.cs

```diff
@@ -784,7 +784,7 @@ public IEnumerable<UnspentOutputReference> GetSpendableTransactionsInAccount(Wal
                         $"Account '{walletAccountReference.AccountName}' in wallet '{walletAccountReference.WalletName}' not found.");
                 }
 
-                res = account.GetSpendableTransactions(this.chain.Tip.Height, confirmations).ToArray();
+                res = account.GetSpendableTransactions(this.chain.Tip.Height, consensus, confirmations, includeImmature: includeImmature).ToArray();
```

Comment:
Not sure if explicitly naming the argument here has any value

Reply:
Elsewhere I have explicitly named it because in 99% of cases you do not want to use anything other than `false` for it, so using it at all is unexpected and it should be visually apparent what parameter is being passed. However, since an argument with the same name is being passed in this case, it does not really give additional information. I will remove it.